templates/caddy.conf

```diff
@@ -2,6 +2,9 @@
 # version: fea0d5a
 # changes:
 # * Set user and group via ansible
+# * Set binary dir via Ansible
```

Comment:
How about merging these into one comment? Since Ansible variables are quite easy to see we don't have to be too specific here